File: C++/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P08_NumberSequence.cpp
#include <iostream>
#include <climits>
#include <algorithm>
using namespace std;

int main() {

	int maxNum = INT16_MIN;
	int minNum = INT16_MAX;

	int n;
	cin >> n;

	for (int i = 0; i < n; i++) {
		int input;
		cin >> input;

		maxNum = max(maxNum, input);
		minNum = min(minNum, input);
	}

	cout << "Max number: " << maxNum << endl;
	cout << "Min number: " << minNum << endl;

	return 0;
}

File: Python/M01_ProgrammingBasics/L04_ForLoop/Lab/Solutions/P08_NumberSequence.py
import sys

max_num = -sys.maxsize
min_num = sys.maxsize
current = 0

numbers = int(input())

for each in range(numbers):
    current = int(input())

    if current > max_num:
        max_num = current
    if current < min_num:
        min_num = current

print(f'Max number: {max_num}')
print(f'Min number: {min_num}')